### PR TITLE
Project 2: The Next Web Clone

### PR DESCRIPTION
**Pull request by tmampa**
In this project 

> We made the nav bar with flexbox
> Footer with flexbox
> Used media queries for responsiveness
> Added breakpoints for 2 other types of display

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+              <li><a href="#">About</a></li>
+              <li><a href="#">Advertise</a></li>
+              <li><a href="#">Jobs</a></li>
+              <li><a href="#">Contact</a></li>
+              </ul>
+          </nav>
+            </div>
+    
+            <div class="flex mid-nav">
+              <a class="logo-img" >
```
Comment (henatan99) [inline index.html:36]: You forgot href value inside this a - tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+            </div>
+    
+            <div class="flex btm-nav">
+          <a href="#"><p>Calling young digital talent in the Netherlands - Be recognized in the T500 →</p></a>
+              </div>
+        </nav>
+      </header>
+<main>
+  <section class="showcases flex">
+    <div class="showcase image-1 flex ">
```
Comment (henatan99) [inline index.html:59]: This div element better b article for the sake of semantics. Use the same comment for the other articles in the showcase section

File: index.html [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+        <li><i class="far fa-clock"></i> 2 hours ago</li>
+        <li><i class="fas fa-share-alt"></i> 5</li>
+      </ul>
+      </div>
+    </div>
+    </section>
+
+    <section class="latest-funding wrapper flex">
+      <div class="sec-heading flex">
+        <h2>Latest funding rounds on Index.co  	&gt;</h2>
```
Comment (henatan99) [inline index.html:91]: This title heading element should also have a link (a-tag)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+        <ul class="flex">
+        <li><a href="#">Add info</a></li>
+        <li id="updates" ><a href="#">Get updates by email</a></li>
+        <li><a href="#">See more</a></li>
+        </ul>
```
Comment (henatan99) [inline index.html:96]: Wrap this ul element in a nav

**Review by henatan99 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
Hi there, you did an excellent work :+1: :clap: :dancers: 
Yet you have a few changes before your project gets approved. See the below-summarized comments and inline comments in the PR. 
### Reminder: - This is your first review and you will be left with a maximum of two more reviews (re-reviews). If your project doesn't get approved at the third review, you will be asked to repeat the project from the beginning. 

## To Highlight 
- Your project fulfills most of the requirements at first CRR 
- Your project captures the original design very well

## To Improve [OPTIONAL]
- Improve semantic tag use in article elements (you have to use div element where you should have used article elements). Please see the inline comments 
- Refer the other inline comments for mandatory improvement of your project 

### Please, work on the requested changes and make another core review request.

### Reminder: - This is your first review and you will be left with a maximum of two more reviews (re-reviews). If your project doesn't get approved at the third review, you will be asked to repeat the project from the beginning.

**Review by MkrtichSargsyan — APPROVED** [review]
## _Status:_ Approved  :hamster: :rocket:
Hi Guys.

Congratulations on your hard work, your project is now approved, 
You can go ahead and merge this branch to the base branch of this repo.

Good luck in your future projects and happy coding :relaxed:

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

**Pull request merged**